### ISSUE TITLE
Avoid cloudfront distribution to be deployed with every `terraform apply`

### ISSUE THREAD

**Issue by jgeluk** [RJPearson94/terraform-aws-open-next#8]
## Technical Details

- OS: Ubuntu
- Terraform Version: 1.6.1
- Provider Version: latest
- Module Version: 5.24.0

## Describe the bug

With every `terraform apply` the cloudfront distribution gets updated which takes a long time.
It seems to be caused by the fact that the `web_acl_id` is not explicitly set, so it gets set to `null` while there's always a value for it, causing an update:

```
 # module.open-next.aws_cloudfront_distribution.website_distribution will be updated in-place
  ~ resource "aws_cloudfront_distribution" "website_distribution" {
        id                             = "E7456xxxMV375"
        tags                           = {}
      - web_acl_id                     = "1dde0680-xxxx-xxxx-ba11-258e6b391ced" -> null
        # (19 unchanged attributes hidden)

        # (11 unchanged blocks hidden)
    }
```

## Steps to Reproduce

Steps to reproduce the behaviour:

1. `terraform apply`
2. `terraform apply`
3. See error

## Expected behaviour

No updates when there are no updates.

**Comment by RJPearson94** [comment]
Hi @jgeluk,

Apologies you ran into the issue, you can now supply web_acl_id as a variable, this should prevent the drift from occurring. This has now been added to the module and released as part of v1.4.0.
Please update your terraform configuration to include the following

```tf
module "open-next" {
  source  = "RJPearson94/open-next/aws"
  version = "1.4.0"
}
```

Thanks for using the module

Kind Regards
Rob

**Comment by jgeluk** [comment]
Thanks that worked, deploys are fast again!
